src/generic/HelixImporter.js

```diff
@@ -164,11 +172,12 @@ class HelixImporter {
             let response;
             try {
               response = await rp({
-                uri: src,
+                uri: getIPURL(src),
```

Comment:
does this work? don't you need to set the host header then?

Reply:
Import worked multiple times since this code was introduced. But it does not work locally, adding the Host header fixes the issue.
Since the DNS change has not been made, it is impossible to test. Will need to keep an eye on it.